components/Footer.js

```diff
@@ -1,14 +1,46 @@
 import React from 'react';
+import FooterLink from './FooterLink';
 
-const Footer = () => {
+const FooterWrapper = (props) => {
+  const style = {
+    backgroundColor: '#D2D3DC',
+    border: 'solid 1px #D2D3DC',
+    boxShadow: '0 0 5px #777',
+    padding: 15,
```

Comment:
React has a convention that allows for 'px' values to be entered as an integer instead of a string.

So, for any un-typed CSS values,, if they're integers, definitely assume that it means 'px', since that's how they'll be treated.

(As to why this convention is exists: ¯\\\_(ツ)_/¯ )